CMakeLists.txt

```diff
@@ -52,16 +52,16 @@ else()
 	add_test(NAME crow_test COMMAND ${CMAKE_CURRENT_BINARY_DIR}/tests/unittest)
 	add_test(NAME template_test COMMAND ${CMAKE_CURRENT_BINARY_DIR}/tests/template/test.py WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/tests/template)
 
-	file(MAKE_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/amalgamate)
+	file(MAKE_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/scripts)
 
-	add_custom_command(OUTPUT ${CMAKE_CURRENT_BINARY_DIR}/amalgamate/crow_all.h
-		COMMAND python ${PROJECT_SOURCE_DIR}/amalgamate/merge_all.py
+	add_custom_command(OUTPUT ${CMAKE_CURRENT_BINARY_DIR}/scripts/crow_all.h
```

Comment:
Maybe just output resulting header to build dir?

Reply:
right, will do